padinfo/padinfo.py

```diff
@@ -1409,3 +1410,20 @@ async def expcurve(self, ctx, start, end: Optional[int], *, query=''):
         state = ClosableEmbedViewState(original_author_id, ClosableEmbedMenu.MENU_TYPE, query,
                                        query_settings, ExperienceCurveView.VIEW_TYPE, props)
         await menu.create(ctx, state)
+
+    @commands.command()
+    async def board(self, ctx, *, message):
+        """Generate a Dawnglare link from the user provided string.
+        Rows/columns separated with spaces.
+        Default fill direction is right then down.
+        Use -1 to invert fill, going down then right."""
+
+        board = BoardGenerator(message)
```

Comment:
let's uppercase the message before parsing it so that people can input lowercase letters and be lazy